### PR TITLE
Clarify multi-selection focus behavior

### PR DESCRIPTION
**Pull request by youknowriad**
Just some follow-up comments to #11237 to clarify the intentions.

### CODE REVIEW

File: packages/editor/src/components/block-list/block.js [hunk 1]
```diff
@@ -94,8 +94,9 @@ export class BlockListBlock extends Component {
 			this.focusTabbable( true );
 		}
 
-		// When triggering a multi-selection,
-		// move the focus to the wrapper of the first selected block.
+		// When triggering a multi-selection, smove the focus to the wrapper of the first selected block.
```
Comment (gziolo) [inline packages/editor/src/components/block-list/block.js]: Typos in `smove`

**Review by gziolo — APPROVED** [review]
Travis is very sensitive to inline comment changes 🤣 

Looks good to me, you can merge when typo is fixed. You can also wait for @aduth as I better understand why we introduced this change so I'm biased.

**Comment by aduth** [comment]
I'd rather wait for #10699 to avoid more merge conflicts.

**Comment by gziolo** [comment]
@youknowriad how about you rebase and merge it? #10699 needs rebase anyway 😃

**Comment by gziolo** [comment]
Rebased, typo fixed and landed this PR.

**Pull request merged**